### PR TITLE
[bitnami/kafka] Add Kafka Helm Chart tests

### PR DESCRIPTION
**Pull request by superaleks**
**Description of the change**
Increasing the test coverage of the Kafka Helm Chart by adding Goss tests. 

**Benefits**

Ensuring higher quality of the Helm charts.

**Possible drawbacks**
Automated tests could be potentially flaky. We took a lot of precautions for this not to be the case, but there is always a risk. 

**Test results**

:gear: - [PASS](https://github.com/superaleks/charts/pull/33)

### Checklist

- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in the agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by FraPazGal — DISMISSED** [review]
Thanks for the PR @superaleks, LGTM!

File: .vib/kafka/goss/goss.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+command:
+  create-kafka-topic:
+    exec: /opt/bitnami/kafka/bin/kafka-topics.sh --create --topic quickstart-events$RANDOM --bootstrap-server localhost:9092
```
Comment (joancafom) [inline .vib/kafka/goss/goss.yaml:3]: Actually, you could use:

```suggestion
    exec: kafka-topics.sh --create --topic quickstart-events$RANDOM --bootstrap-server localhost:9092 
```
and that would additionally check if `kafka-topics.sh` is in the PATH (which should be).

**Review by joancafom — COMMENTED** [review]
Thanks for your PR! Could you please take a look at my comments?

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for the PR @superaleks 🚀

**Pull request merged**